src/profile-logic/active-tab.js

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I had to create a new file for them, since they were creating a cyclic dependency on other files.